src/main/java/uk/gov/companieshouse/api/accounts/ApplicationConfiguration.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.companieshouse.api.accounts;
+
+import com.mongodb.MongoClientOptions;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+
+/**
+ * General application configuration .
+ */
+@Configuration
+@SuppressWarnings("unused")
```

Comment:
This isn't needed. There isn't anything unused.

Reply:
The `mappingMongoConverter` is unused.

Reply:
Same thing as above but with `mongoClientOptions`

Reply:
I opened this in Eclipse and it underlined the "unused" as not needed. When I removed the suppression I had no checkstyle warnings.

Reply:
It does the opposite in IntelliJ haha. I'll remove it then.